examples/server/README.md

```diff
@@ -357,6 +361,16 @@ node index.js
 
     `frequency_penalty`: Repeat alpha frequency penalty. Default: `0.0`, which is disabled.
 
+    `dry_multiplier`: Set the DRY (Don't Repeat Yourself) repetition penalty multiplier. Default: `0.0`, which is disabled.
+
+    `dry_base`: Set the DRY repetition penalty base value. Default: `1.75`
+
+    `dry_allowed_length`: Tokens that extend repetition beyond this receive exponentially increasing penalty: multiplier * base ^ (length of repeating sequence before token - allowed length). Default: `2`
+
+    `dry_penalty_last_n`: How many tokens to scan for repetitions. Default: `-1`, where `0` is disabled and `-1` is context size.
+
+    `dry_sequence_breakers`: Specify an array of sequence breakers for DRY sampling. Can be provided as a JSON array of strings or as a JSON-encoded string representing an array of strings. Default: `["\n", ":", "\"", "*"]`
```

Comment:
There was a discussion above mentioning a format incompatible with TGWUI that SillyTavern would need to accommodate, but the options described here (which include "a JSON-encoded string representing an array of strings") appear to be compatible with existing frontends. Has the proposed change not been implemented yet?

Reply:
The change was in fact made but this had not been updated yet to reflect that. Thank you for catching this.